core/modules/tests.rs

```diff
@@ -275,7 +288,8 @@ impl ModuleLoader for MockLoader {
 
   fn code_cache_ready(
     &self,
-    module_specifier: &ModuleSpecifier,
+    module_specifier: ModuleSpecifier,
+    _hash: u64,
     code_cache: &[u8],
   ) -> Pin<Box<dyn Future<Output = ()>>> {
```

Comment:
We should look into doing something similar for the script callbacks. I'm just not sure how to convert a `Local<String>` to a `&str`... maybe deno_core should just create the hash.

Reply:
You can convert it like so: `&x.to_rust_string_lossy(scope)`